src/connector/src/kafka/enumerator/client.rs

```diff
@@ -214,6 +214,10 @@ impl KafkaSplitEnumerator {
             _ => return Err(anyhow!("topic {} not found", self.topic)),
         };
 
+        if topic_meta.partitions().is_empty() {
```

Comment:
```
     let topic_meta = match metadata.topics() {
            [meta] => meta,
            _ => return Err(anyhow!("topic {} not found", self.topic)),
        };
```

Can we delete the previous lines?

Reply:
Fine, but the code may not look very good